source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -106,9 +106,22 @@ namespace RendererQml
         AddContainerElements(bodyLayout, card->GetBody(), context);
         AddActions(bodyLayout, card->GetActions(), context);
 
+		bodyLayout->Property("onHeightChanged", Formatter() << "{generateStretchHeight(bodyLayout.children," << card->GetMinHeight() << ");}");
+
+		bodyLayout->Property("onWidthChanged", Formatter() << "{generateStretchHeight(bodyLayout.children," << card->GetMinHeight() << ")}");
```

Comment:
`generateStretchHeight` -> is this name correct here? you are check for `onWidthChanged`.

Reply:
Yes, Had to apply generateStretchHeight() for both onHeightChanged and onWidthChanged. And since the bodyLayout is not column set generateStretchWidth() is not required